data/binding/binding_test.go

```diff
@@ -22,7 +22,6 @@ func TestBase_AddListener(t *testing.T) {
 	assert.Equal(t, 1, len(data.listeners))
 
 	waitForItems()
-	data.trigger()
```

Comment:
I’m curious about this change, can you help?

Reply:
Yes, `data.AddListener(fn)` already enqueues DataChanged callback, so trigger it again is not needed, is it?

Reply:
That looks to be the case